vars/runPipeline.groovy

```diff
@@ -103,9 +103,16 @@ def retryOnPreemption (action, times = 1) {
         try {
```

Comment:
Don't you want to set `times` higher than `1`?
If I understand it correctly it wouldn't do the restart even if the exception is properly.

Reply:
renamed to `retries` for clarification

Reply:
The last attempt rethrows https://github.com/dlang/ci/pull/124/files#diff-e15b7462050c1530715a62b67a9b7e02R118.

Reply:
Ah okay. I would still increase it it 2 or 3 as I have seen these exceptions rather frequently in the last weeks.

Reply:
Yeah, despite opposite claims, GCE is quite aggressively preempting instances. Maybe I need to switch datacenters.